apps/openmw/mwmechanics/pathfinding.cpp

```diff
@@ -51,8 +51,7 @@ namespace
                                                   const MWWorld::CellStore *cell,
                                                   const osg::Vec3f pos, int start)
     {
-        if(!grid || grid->mPoints.empty())
-            return std::pair<int, bool> (-1, false);
+        assert(grid && !grid->mPoints.empty());
```

Comment:
This change breaks consistency with getClosestPoint. Either make this change in both functions or change none of them.